pkg/mod/util.go

```diff
@@ -0,0 +1,44 @@
+package mod
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func runGo(ctx context.Context, stdout io.Writer, args ...string) error {
```

Comment:
be careful with variable names... stdout and stderr have specific meaning. this var name should be `out`, and stderr below shuold probably be something like `errbuf`.

Also, I'm not sure this should be writing to os.Stderr at all, I'd hook cmd.Stderr up too stdout